Services/CO.CDP.EntityVerification/CO.CDP.EntityVerification.csproj

```diff
@@ -22,12 +22,13 @@
           <PrivateAssets>all</PrivateAssets>
           <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
-        <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.2" />
+        <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.4" />
         <PackageReference Include="Amazon.Sqs" Version="0.33.0" />
         <PackageReference Include="AWSSDK.SQS" Version="3.7.301.31" />
         <ProjectReference Include="..\..\Libraries\CO.CDP.Functional\CO.CDP.Functional.csproj" />
         <ProjectReference Include="..\..\Libraries\CO.CDP.Swashbuckle\CO.CDP.Swashbuckle.csproj" />
         <ProjectReference Include="..\..\Libraries\CO.CDP.MQ\CO.CDP.MQ.csproj" />
+        <ProjectReference Include="..\CO.CDP.OrganisationInformation.Persistence\CO.CDP.OrganisationInformation.Persistence.csproj" />
```

Comment:
Entity verification should be an independent component. If we can, we should not introduced dependencies shared with other projects. Why do we nee OrganisationInformation.Persistence here?

Reply:
I think I understand this was done because of IEntityDate. To prevent the above dependency, I'm moving it out to a dedicated library: https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/pull/324